packages/twenty-server/src/workspace/workspace-migration-runner/workspace-migration-runner.service.ts

```diff
@@ -298,6 +298,7 @@ export class WorkspaceMigrationRunnerService {
         columnNames: [migrationColumn.columnName],
         referencedColumnNames: [migrationColumn.referencedTableColumnName],
         referencedTableName: migrationColumn.referencedTableName,
+        referencedSchema: schemaName,
```

Comment:
we should add this in case we use a single datasource as we need to make sure the migration are applied to the right schema